protocol/v2/ssv/runner/aggregator.go

```diff
@@ -137,27 +137,19 @@ func (r *AggregatorRunner) ProcessPreConsensus(ctx context.Context, logger *zap.
 		return fmt.Errorf("got pre-consensus quorum but it has invalid signatures: %w", err)
 	}
 
-	signer := ssvtypes.PartialSigMsgSigner(signedMsg)
 	duty := r.state().CurrentDuty.(*spectypes.ValidatorDuty)
 	span.SetAttributes(
 		observability.CommitteeIndexAttribute(duty.CommitteeIndex),
 		observability.ValidatorIndexAttribute(duty.ValidatorIndex),
 	)
 
-	const gotPartialSignaturesEvent = "🧩 got partial aggregator selection proof signatures"
```

Comment:
Maybe it'll be smart to log who participated in the pre consensus quorum of we eventually happen to be aggregators